python/lsst/pipe/tasks/calibrate.py

```diff
@@ -598,6 +734,10 @@ def run(self, exposure, exposureIdInfo=None, background=None,
             sourceCat=sourceCat,
             astromMatches=astromMatches,
             matchMeta=matchMeta,
+
```

Comment:
Just seems like a weird place for a newline.  I'd either drop it or replace it with comment headings if you want to define groups of output struct fields.